inbm/fpm/mqtt/template/etc/intel-manageability/public/mqtt-broker/acl.file

```diff
@@ -54,6 +54,11 @@ topic write manageability/response
 topic write manageability/telemetry
 topic read manageability/cmd/+
 
+user ucc-native-service
+topic write manageability/response
+topic write manageability/telemetry
+topic read manageability/cmd/+
```

Comment:
the channel would be manageablity/request/command for the last line.  I believe that's correct anyways.  Everything that comes into cloudadapter comes in on manageability/request/{extra topic} which I made that command.